img/foxsi_image_det.pro

```diff
@@ -14,6 +14,7 @@
 ;				Default entire flight
 ;		THR_N	Threshold to use for n-side data.
 ;		YEAR	2012 or 2014 flight.  Default 2014.
```

Comment:
Is YEAR 2018 also available?

Reply:
It is not yet. My plan was to introduce all the 2018 keywords in a later pull request, once this one is accepted, but if you think I should introduce it now it is possible.